packages/nimble-components/src/table/template.ts

```diff
@@ -55,6 +56,6 @@ export const template = html<Table>`
                 </div>
             </div>
         </div>
-        <slot ${slotted({ property: 'columns', filter: isTableColumn() })}></slot>
+        <slot></slot>
```

Comment:
Is this line still required? Since named slots are being created above for each column, I wouldn't think we still need this unnamed slot.

Reply:
Removed.